packages/clarity-js/src/data/metadata.ts

```diff
@@ -36,10 +36,11 @@ export function start(): void {
   dimension.log(Dimension.PageLanguage, document.documentElement.lang);
   dimension.log(Dimension.DocumentDirection, document.dir);
   dimension.log(Dimension.DevicePixelRatio, `${window.devicePixelRatio}`);
-  
+  dimension.log(Dimension.Dob, u.dob);
```

Comment:
does this need a null check

Reply:
The function already checks for null values first.